sectionproperties/pre/library/concrete_sections.py

```diff
@@ -25,6 +26,9 @@ def concrete_rectangular_section(
     :param int n_bar: Number of steel reinforcing bars
     :param int n_circle: Number of points discretising the steel reinforcing bars
     :param float cover: Side and bottom cover to the steel reinforcing bars
+    :param float area: If provided, constructs reinforcing bars based on their area
+        rather than a diameter (prevents the underestimation of steel area due to
+        circle discretision)
```

Comment:
@robbievanleeuwen small spelling mistake here, should be ```discretisation``` ?

Similar on line 122, 212, etc

Reply:
Cheers for the pickup @Agent6-6-6

Reply:
Fixed with d52dda6.